### DISCUSSION TITLE
Welcome to pdnext Discussions!

### DISCUSSION THREAD

**Discussion by sebshader** [sebshader/pdnext#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by IvanPedroMuricy** [comment]
Greetings, I use pdnext in Windows and really enjoy the gui colors options. However, I am not being able to install it in Arch Linux. I follow all the steps, but, in the end, all I get is the Vanilla version. I would welcome any suggestions. Also, if you had any installation tutorial in YouTube, this would be great.
Best wishes.

Ivan

**Reply by sebshader** [reply]
Hi,
In order to install on Linux you have to compile it yourself. This is because every distribution has different packaging, glibc etc.
Feel free to post any compilation issues
Edit: hereis a configure command people have had luck with:
```
./configure --enable-alsa --enable-jack --enable-portaudio --enable-portmidi CFLAGS=""
```
(You have to install the right headers etc.)

If you have successfully compiled and installed then make sure the color files/plugins are in the right directory..
